### DISCUSSION TITLE
Playbook elastalert rules are not working after updating rules from sigma repo

### DISCUSSION THREAD

**Discussion by peas100** [Security-Onion-Solutions/securityonion#5973] [General] [answered]
Hi I am new to security onion playbook feature. I am trying to update the Sigma rules. As per documentation it has referred sigma rules from https://github.com/Neo23x0/sigma/tree/master/rules.
However when trying to do `so-playbook-ruleupdate` there are no new rules fetched. I checked `playbook_bulk-update.py`,
it is pulling from https://github.com/Security-Onion-Solutions/sigma.git.
Above repo is not updated since Nov 2020.

So I modified the URL. (Is this correct way?)
New plays are added, however the elastalert rule created is not matching with data (Rule is quite different than the rule being created using uncoder.io).

In playbook.py there are few backend options used while converting. Does that need update?
Could anyone let me know if there is something missing?

**Comment by defensivedepth** [comment, marked as answer]
There is ongoing work around this feature - I will update this Discussion with more details once we have it figured out.

**Reply by peas100** [reply]
Ok. Any update on this ?

**Reply by defensivedepth** [reply]
Not at this time.
